full-moon/src/ast/parsers.rs

```diff
@@ -1285,6 +1285,25 @@ cfg_if::cfg_if! {
                 state = new_state;
             }
 
+            // if let Ok((state, pipe)) = ParseSymbol(Symbol::Pipe).parse(state) {

```

Comment:
Don't leave commented code in version control, just remove it.

Reply:
I completely missed this, will remove